### PR TITLE
feat(button): ✨  add button group

### PR DESCRIPTION
**Pull request by navin-moorthy**

### CODE REVIEW

File: src/icons/ArrowNarrowRight.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as React from "react";
```
Comment (navin-moorthy) [inline src/icons/ArrowNarrowRight.tsx:1]: Using in another project requires the react import.

File: src/utils/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { cx } from "@renderlesskit/react";
 import { overrideTailwindClasses } from "tailwind-override";
 
-import tailwindProperties from "../../tailwindProperties.json";
+import { tailwindProperties } from "./tailwindProperties";
```
Comment (navin-moorthy) [inline src/utils/index.tsx:4]: These properties should be part of the build inorder to utilize them.

File: src/button/stories/Button.stories.tsx [hunk 3]
```diff
@@ -96,3 +97,73 @@ CLButton.args = {
...
+);
+
+export const GroupStyled = BGSDefault.bind({});
+GroupStyled.args = {
+  isAttached: true,
+  size: "lg",
+  variant: "secondary",
+};
+
+const BGLDefault: Story<ButtonGroupProps> = args => {
```
Comment (anuraghazra) [inline src/button/stories/Button.stories.tsx:142]: Can we change all the story names? they are kind of cryptic.

File: src/button/Button.tsx [hunk 4]
```diff
@@ -43,8 +44,8 @@ function ButtonComponent(
   ref: React.Ref<HTMLButtonElement>,
 ) {
   const {
-    size = "md",
-    variant = "primary",
+    size,
+    variant,
     leftIcon,
     rightIcon,
```
Comment (anuraghazra) [inline src/button/Button.tsx:50]: Since we are here let's also change these to prefix and suffix

Comment (navin-moorthy) [reply src/button/Button.tsx:50]: Yeas.

**Review by anuraghazra — APPROVED** [review]

**Comment by anuraghazra** [comment]
LGTM!

**Pull request merged**